wikidatarefisland/pipes/value_matcher_pipe.py

```diff
@@ -0,0 +1,34 @@
+from wikidatarefisland.data_model.wikibase import ValueMatchers
+from wikidatarefisland.pipes import AbstractPipe
+
+
+class ValueMatcherPipe(AbstractPipe):
```

Comment:
I'd say please rename it to SimpleValueMatcherPipe, we are going to have more than one ValueMatcherPipe

Reply:
See my comment above